namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-dev/resources/probation-in-court-cases-queue.tf

```diff
@@ -0,0 +1,114 @@
+module "probation-in-court-cases-queue" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=5.0.0"
+
+  # Queue configuration
+  sqs_name                    = "probation-in-court-cases"
```

Comment:
The generated name of this queue is
probation-in-court-team-development-probation-in-court-cases-dead-letter-queue.fifo

The build considers this to be invalid, and I think it is because it is too long - the limit is 80 characters.

Let's follow the convention used in the topics, which is a bit shorter. If you just set the `sqs_name` to `court-cases` that should do the trick